Frameworks/UIKit/UITextField.mm

```diff
@@ -93,18 +72,14 @@ @implementation UITextField {
     BOOL _adjustsFontSizeToFitWidth;
 
     // backing xaml textbox and passwordBox
-    StrongId<UIView> _subView; // Container UIView for the Xaml textbox or passwordbox
+    StrongId<WXFrameworkElement> _textField; // Backing user control
     StrongId<WXCTextBox> _textBox; // Backing xaml textbox
     StrongId<WXCPasswordBox> _passwordBox; // Backing xaml passwordbox
     StrongId<WXCControl> _textContentElement; // ContentElement from the TextBox template to calculate width
     StrongId<WXCControl> _passwordContentElement; // ContentElement from the PasswordBox template to calculate width
```

Comment:
I don't think we should need these here.  Can't we push all of this into the Xaml control implementation?